funnel/templates/macros.html.jinja2

```diff
@@ -706,3 +706,54 @@
     </div>
   </div>
 {%- endmacro %}
+
+{%- macro open_submission(project, showEditBtn=true) %}
+  <div class="flex-wrapper flex-wrapper--space-between flex-wrapper--start margin-top bottom-padding">
+    <div>
+      <div class="js-cfp-status {% if not project.cfp_state.OPEN -%} mui--hide {%- endif %}">
+        <p class="mui--text-subhead mui--text-bold margin-bottom" data-cy="cfp-state">{% trans %}Submission status: ON{% endtrans %}</p>
```

Comment:
This text could be better. Is it necessary given "Accepting submissions" right below?